### DISCUSSION TITLE
kimai2_sessions table

### DISCUSSION THREAD

**Discussion by miomirnoubis** [kimai/kimai#3300] [Questions & Answers] [answered]
Hello,
I've noticed recently that kimai2_sessions table suddenly jumped in size and became rather large, so just wanted to check if it is safe to clean it (with truncate command).
Thanks in advance for the answer.

**Comment by kevinpapst** [comment, marked as answer]
No problem, but the table will shrink its size automatically at some random point in time. 
Emptying the table is no problem, the downside of that is that you will log out all active users.

**Reply by miomirnoubis** [reply]
Thank you very much for the quick answer.
